src/ia-dropdown.ts

```diff
@@ -22,6 +22,26 @@ export class IaDropdown extends LitElement {
 
   @property({ type: Boolean, attribute: true }) displayCaret = false;
 
+  /**
+   * To allow customizing whether the dropdown menu automatically closes
+   * when an option is selected.
+   */
+  @property({ type: Boolean, attribute: true }) closeOnSelect = false;
```

Comment:
the dropdown currently closes on select, can we make default to `true` so that current consumers do not have to change their props?

Reply:
I believe current consumers have to close the dropdown manually (e.g., see https://github.com/internetarchive/iaux-music-player/blob/main/src/channel-selector/channel-selector.ts#L137).

So I think a default of `false` here is correct for backwards-compatibility?